scripts/build.sh

```diff
@@ -10,6 +10,11 @@ echo "Executing in $(pwd)"
 echo "Description: Build the package in build/"
 echo "====================================================="
 
+echo "Installing Dependencies"
+echo "Note: if EEXIST: file already exists on tsc, run `npm uninstall -g tsc`"
+npm install -g typescript
```

Comment:
Do we need to install typescript globally here?

Reply:
I added it to ensure we have tsc available as CLI tool

Reply:
Isn't this only called by the CI step though? I see that devcontainer already installs typescript CLI globally.

Reply:
Ok, removed it